tests/tck/features/match/VariableLengthPattern.feature

```diff
@@ -251,19 +251,19 @@ Feature: Variable length Pattern match (m to n)
       RETURN e
       """
     Then the result should be, in any order, with relax comparison:
-      | e                                           |
-      | [[:like "Tim Duncan"->"Manu Ginobili"]]     |
-      | [[:like "Tim Duncan"->"Tony Parker"]]       |
-      | [[:like "Tim Duncan"<-"Dejounte Murray"]]   |
-      | [[:like "Tim Duncan"<-"Shaquille O'Neal"]]  |
-      | [[:like "Tim Duncan"<-"Marco Belinelli"]]   |
-      | [[:like "Tim Duncan"<-"Boris Diaw"]]        |
-      | [[:like "Tim Duncan"<-"Manu Ginobili"]]     |
-      | [[:like "Tim Duncan"<-"Danny Green"]]       |
-      | [[:like "Tim Duncan"<-"Tiago Splitter"]]    |
-      | [[:like "Tim Duncan"<-"Aron Baynes"]]       |
-      | [[:like "Tim Duncan"<-"Tony Parker"]]       |
-      | [[:like "Tim Duncan"<-"LaMarcus Aldridge"]] |
+      | e                                         |
```

Comment:
before
```
-----+-----------+--------------+----------------+-------------------------------------------------
| id | name      | dependencies | profiling data | operator info                                  |
-----+-----------+--------------+----------------+-------------------------------------------------
| 10 | Project   | 9            |                | outputVar: {                                   |
|    |           |              |                |   "colNames": [                                |
|    |           |              |                |     "e"                                        |
|    |           |              |                |   ],                                           |
|    |           |              |                |   "name": "__Project_6",                       |
|    |           |              |                |   "type": "DATASET"                            |
|    |           |              |                | }                                              |
|    |           |              |                | inputVar: __Traverse_3                         |
|    |           |              |                | columns: [                                     |
|    |           |              |                |   "[__VAR_1 IN $-.e WHERE is_edge($__VAR_1)]"  |
|    |           |              |                | ]                                              |
-----+-----------+--------------+----------------+-------------------------------------------------
|  9 | Traverse  | 7            |                | outputVar: {                                   |
```

now:
```
-----+-----------+--------------+----------------+-------------------------------------------------
| id | name      | dependencies | profiling data | operator info                                  |
-----+-----------+--------------+----------------+-------------------------------------------------
| 10 | Project   | 9            |                | outputVar: {                                   |
|    |           |              |                |   "colNames": [                                |
|    |           |              |                |     "e"                                        |
|    |           |              |                |   ],                                           |
|    |           |              |                |   "type": "DATASET",                           |
|    |           |              |                |   "name": "__Project_6"                        |
|    |           |              |                | }                                              |
|    |           |              |                | inputVar: __Traverse_3                         |
|    |           |              |                | columns: [                                     |
|    |           |              |                |   "$-.e[0]"                                    |
|    |           |              |                | ]                                              |
-----+-----------+--------------+----------------+-------------------------------------------------
|  9 | Traverse  | 7            |                | outputVar: {                                   |
```


If Match‘s STEP is  `1..1 OR *1`, it was edge List type before, and it is edge type now